### PR TITLE
Add documentation to create a new entity

### PR DESCRIPTION
**Pull request by maximehuran**
- Declare entity
- Create interfaces and classes
- Declare Sylius resource

### CODE REVIEW

File: create-new-entity/declare-your-entity.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Declare your entity 
+
```
Comment (jacquesbh) [inline create-new-entity/declare-your-entity.md:2]: What's en entity? in which cases do we use one? etc.

File: create-new-entity/create-entity-classes.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+     * @return TranslationInterface
+     */
+    protected function createTranslation(): TranslationInterface
+    {
+        return new JobTranslation();
+    }
+}
+```
+
+For all translation's attributes , we use the function `getJobTranslation` which use `getTranslation` of `TranslatableTrait`.
```
Comment (jacquesbh) [inline create-new-entity/create-entity-classes.md:231]: Why not using `getTranslation` directly?

File: create-new-entity/declare-sylius-resource.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+In my case, it is : 
+```yml
+    # [...]
+    - { resource: '@AppBundle/Resources/config/app/config.yml' }
+    # [...]
+```
+
+Then, in your `AppBundle` config file, add this line : 
+```yml
+imports:
```
Comment (jacquesbh) [inline create-new-entity/declare-sylius-resource.md:15]: Add `# src/AppBundle/Resources/config/app/config.yml`

File: create-new-entity/declare-sylius-resource.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        $this->addSql('DROP TABLE app_job');
+        $this->addSql('DROP TABLE app_job_channels');
+        $this->addSql('DROP TABLE app_job_translation');
+    }
+}
+```
+
+If you check in your database, tables are not created yet. To create it, use : 
+```bash
+console doctrine:schema:update --force
```
Comment (jacquesbh) [inline create-new-entity/declare-sylius-resource.md:108]: Nop.

`console doctrine:migration:migrate` is better :)

File: create-new-entity/create-entity-classes.md [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+
+    public function setDescription(?string $description): void
+    {
+        $this->description = $description;
+    }
+}
+
+```
+
+You can try now your doctriner migration. And you will see...
```
Comment (ghost) [inline create-new-entity/create-entity-classes.md:301]: `doctriner` => `doctrine`.

**Review by fplantinet — APPROVED** [review]
ok

**Pull request closed without merging**